.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules/
+coverage/
 persons.json
+persons2.json
+fileOperations.test.js
```

Comment:
This file shouldn't be on gitignore because it's a test, right? The next developer that works on this will need it, right?